diff -up konquest-20.03.90/dialogs/newgamedlg.cc.omv~ konquest-20.03.90/dialogs/newgamedlg.cc
--- konquest-20.03.90/dialogs/newgamedlg.cc.omv~	2020-04-13 23:57:16.801736542 +0200
+++ konquest-20.03.90/dialogs/newgamedlg.cc	2020-04-13 23:57:28.356865632 +0200
@@ -370,11 +370,7 @@ NewGameDlg::NewGameDlg( QWidget *parent,
     connect(m_w->randomizeMap, &QPushButton::clicked, this, &NewGameDlg::slotNewMap);
     connect(m_w->removePlayerButton, &QPushButton::clicked, this, &NewGameDlg::slotRemovePlayer);
 
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_w->OwnerCB, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this, &NewGameDlg::slotNewOwner);
-#else
-    connect(m_w->OwnerCB, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this, &NewGameDlg::slotNewOwner);
-#endif
     connect(m_w->KillPercentageSB, static_cast<void (QDoubleSpinBox::*)(double)>(&QDoubleSpinBox::valueChanged), this, &NewGameDlg::slotNewKillPercentage);
     connect(m_w->ProductionSB, static_cast<void (QSpinBox::*)(int)>(&QSpinBox::valueChanged), this, &NewGameDlg::slotNewProduction);
 
@@ -503,11 +499,7 @@ NewGameDlg::slotUpdateSelection(const Co
     m_w->KillPercentageSB->setEnabled(hasPlanet);
     m_w->ProductionSB->setEnabled(hasPlanet);
 
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     disconnect(m_w->OwnerCB, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this, &NewGameDlg::slotNewOwner);
-#else
-    disconnect(m_w->OwnerCB, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this, &NewGameDlg::slotNewOwner);
-#endif
 
     if (hasPlanet) {
         Planet *planet = sector->planet();
@@ -526,11 +518,7 @@ NewGameDlg::slotUpdateSelection(const Co
         m_w->OwnerCB->setCurrentIndex(0);
     }
 
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(m_w->OwnerCB, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this, &NewGameDlg::slotNewOwner);
-#else
-    connect(m_w->OwnerCB, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this, &NewGameDlg::slotNewOwner);
-#endif
 }
 
 
